frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -647,6 +655,10 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
                 <dd>
                   {spec.provider && spec.provider.name ? spec.provider.name : 'Not available'}
                 </dd>
+                <dt>Support</dt>
+                <dd>
+                  {ibmSupportLink && <ExternalLink href={ibmSupportLink} text="Get Support" />}
```

Comment:
We should fall back to the standard support link here if available. (Although I think the marketplace support link could use the standard annotation.)